assets/js/selecting.js

```diff
@@ -90,7 +115,12 @@ function redirect() {
             //else if (sslothable(minor.value, nver.value)) { //check for versions that are not cartupdated, cartupdated consoles cannot access the browser, see troubleshooting for solution
             //    window.location.href = "installing-boot9strap-(ssloth-browser)"; //OTHERAPP BROKEN FOR TWN/CHN 11.4+ ATM
             //}
+            else if (minor < 15 && isO3DS) {
+                window.location.href = "installing-boot9strap-(safecerthax)";
```

Comment:
do we actually know if CHN can do safecerthax
like does safe mode actually use safehax here

Reply:
apparently working